### ISSUE TITLE
结果离谱

### ISSUE THREAD

**Issue by lqf0920** [tunghia1890/BERT4Rec_TF2x#1]
global_step = 100000
loss = 6.740822
masked_lm_accuracy = 0.0072363853
masked_lm_loss = 6.7411404

我直接下载代码到本地先运行了data_processing.py，再运行bert4rec_main.py。结果出现了上面的结果，是不是哪里出问题了QAQ
